.github/workflows/test.yml

```
@@ -37,6 +40,22 @@ jobs:
           python -m pip install --upgrade pip setuptools wheel
           python -m pip install -r requirements-test.txt
 
+      - name: Run poetry and check
+        run: |
+          poetry export --without-hashes -o requirements.txt.check
+          poetry export --without-hashes --with dev -o requirements-dev.txt.check
+          poetry export --without-hashes --with test -o requirements-test.txt.check
+          poetry export --without-hashes --with license -o requirements-license.txt.check
+          
+          diff -q requirements.txt requirements.txt.check || \
+          diff -q requirements-dev.txt requirements-dev.txt.check || \
+          diff -q requirements-test.txt requirements-test.txt.check || \
+          diff -q requirements-license.txt requirements-license.txt.check > /dev/null
+          if [$? = 1]; then
```

ここでエラーが起きてそうでした！
```suggestion
          if [ $? = 1 ]; then
```
（普通にエラーが起こってるはずですが、なぜテストが通ったかのような表示になっているんだろう･･･？ 🤔 ）

---

Githubのチェックはテストの実行自体が出来なくても成功した奴だけ出して `All checks have passed`　って出したりしてたので過信は厳禁な気がします（私はたった今過信して見落としましたが……）
修正しておきます